code/game/objects/items/weapons/autolathe_disks.dm

```diff
@@ -1177,6 +1177,23 @@
 		/datum/design/autolathe/cell/medium/high,
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee
+	disk_name = "NeoTheology Armory - Melee Weapons"
+	icon_state = "neotheology"
+	spawn_tags = SPAWN_TAG_DESING_ADVANCED
```

Comment:
All of these weapons cost biomatter, and as such can't be printed at a normal autolathe - only the NT bio-lathe. As such, they're completely worthless for the majority of the crew. Let's keep them from spawning in maint for this reason.
```suggestion
	spawn_blacklisted = TRUE
```

Reply:
Done